client/app/telecom/telephony/billingAccount/billing/repaymentHistory/telecom-telephony-billing-account-billing-repayment-history.controller.js

```diff
@@ -104,26 +77,17 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingRe
     =            INITIALIZATION            =
     ======================================*/
 
-    function init () {
-        self.loading.init = true;
+    this.$onInit = function () {
 
         return TelephonyMediator.getGroup($stateParams.billingAccount).then(function (group) {
             self.group = group;
 
             return fetchHistory().then(function (consumptions) {
-                self.consumption.raw = consumptions;
-                self.sortConsumption();
+                self.consumptionData = consumptions;
             });
-        }).catch(function (error) {
-            Toast.error([$translate.instant("telephony_group_billing_repayment_history_init_error"), (error.data && error.data.message) || ""].join(" "));
-            return $q.reject(error);
-        }).finally(function () {
-            self.loading.init = false;
+        }, function (error) {
+            return new ToastError([$translate.instant("telephony_group_billing_repayment_history_download_error"), (error.data && error.data.message) || ""].join(" "));
```

Comment:
This is not how `ToastError` works. It takes as arguments an error object AND a translation key, not a string